src/components/Square.tsx

```diff
@@ -0,0 +1,12 @@
+interface Props {
+  value: string;
+  onClick(): void;
+}
+function Square(props: Props) {
```

Comment:
I think you can expand props to make it clean

```ts
{value, onClick}
```

And the below call them without `props.`